docs/installation/development.rst

```diff
@@ -104,6 +92,15 @@ Running tests
 See :doc:`../development/testing`
 
 
+Working on the theme
+--------------------
+
+The theme is derived from a separate package: ploneintranet.prototype.
+
+Documentation TODO.
+https://github.com/ploneintranet/ploneintranet.theme/blob/0ba4c21acd6da8e941fa94d0421491e59228009d/README.rst
+
+
 Building the Documentation
 --------------------------
 
```

Comment:
I will port the old theme documentation to also document prototype development.
Am thinking jsrelease belongs in the prototype though. You'd need to commit your changes there anyway first before making a release.